checks/kubernetes/cisbenchmarks/apiserver/security_context_deny_plugin.rego

```diff
@@ -9,6 +9,9 @@
 # custom:
 #   id: KCV0013
 #   avd_id: AVD-KCV-0013
+#   frameworks:
+#     k8s-cis-1.23:
+#       - "1.23"
```

Comment:
Here you need to specify the ID of the control.
```suggestion
#       - "1.2.13"
```

Also `AVD-KCV-0013` is in `k8s-cis-1.24`.

Reply:
thanks! updated.